assets/sass/config/_variables-mui3.scss

```diff
@@ -170,7 +170,7 @@ $c-interactive-tertiary-press: rgba(22, 27, 24, 0.16);
 $c-content-link: #108080;
 $c-content-primary-hover: #2e5f41;
 $c-radio-background-hover: #aeb3b5;
-$c-radio-background-focus: #8a8f90;
+$c-radio-background-focus: #00220d52;
```

Comment:
The design references `interactive/primary-press` which we have above as the same color here, just defined using rgb. See `$c-interactive-primary-press`

<img width="470" alt="image" src="https://user-images.githubusercontent.com/1621608/181383524-2bbdada7-8aac-4a8d-bf78-3fa7319ead66.png">

<img width="470" alt="image" src="https://user-images.githubusercontent.com/1621608/181383512-5fec5d55-d915-4927-837b-a0c13fe6fb83.png">